doc/get_started/index.rst

```diff
@@ -0,0 +1,12 @@
+Getting started
+===============
+
+How to get started using spikeinterface. Also check out spike tutorials!
+
+.. toctree::
+    :maxdepth: 1
+
+    installation
+    import
+    get_started
```

Comment:
actually @h-mayorquin 'quickstart' is good and avoids the redundancy between section name and tutorial name. Maybe 'Quickstart Guide'  or 'Quickside Tutorial' just because 'quickstart' on it's own might be slightly amiguous?